### PR TITLE
Tc 5233 company api v2 docs

### PR DESCRIPTION
**Pull request by olegtradecloud**
## Description

https://tradecloud.atlassian.net/browse/TC-5233

### Scope
This PR create company api manual

### Submitter pre-flight check

- [x] Review your documentation
- [x] Add labels `needs reviewing`
- [x] Assign PR and JIRA ticket to 'Reviewer' (in 'Reviewing' state)

### Reviewer pre-flight check

- [ ] Review documentation in PR
- [ ] Remove label `needs reviewing`
- [ ] Merge the PR and remove the release for this branch from Gitbook
- [ ] Set Jira ticket to 'Released'

### CODE REVIEW

File: company/connect-to-company.md [hunk 1]
```diff
@@ -4,6 +4,28 @@ description: How to connect to another company
...
+{% api-method-body-parameters %} 
+{% api-method-parameter name="otherUserId" type="uuid" required=true %} Request connection to user id {% endapi-method-parameter %}
+{% endapi-method-body-parameters %}
+{% endapi-method-request %}
+{% endapi-method-spec %}
+{% endapi-method %}
+
+## Response
+
+Only a HTTP status code will be returned
```
Comment (roy-tc) [inline company/connect-to-company.md]: Can this somehow be moved to the `Response` tab shown in the page?
And can we add a link to Swagger UI?

Comment (olegtradecloud) [reply company/connect-to-company.md]: Yes I think so. Will fix

Comment (olegtradecloud) [reply company/connect-to-company.md]: @roy-tc 
Not so easy deal with this markup. I can't find good example or description of `api-method`.
So I just added link to swagger and fixed response a bit.

File: company/connect-to-company.md [hunk 2]
```diff
@@ -4,6 +4,34 @@ description: How to connect to another company
...
 
-API
+## Request
+
+{% api-method method="post" host="https://api.accp.tradecloud1.com/v2" path="/company/:companyId/connection/:otherCompanyId/_request"%} 
+{% api-method-summary %} Request company connection {% endapi-method-summary %}
+{% api-method-spec %} 
+{% api-method-request %} 
+{% api-method-headers %} 
+{% api-method-parameter name="Authorization" type="string" required=true %} Authentication token {% endapi-method-parameter %}
```
Comment (marcmatt) [inline company/connect-to-company.md]: Value is: `Bearer <Authentication token>`
Change in all pages

File: company/list-connected-companies.md [hunk 3]
```diff
@@ -3,3 +3,29 @@ description: How to list your connections with other companies
 ---
 
 # List your connections
+Yoc can find all connected companies using `connected=true` parameter
```
[the remark this answers is not in the document]
Comment (olegtradecloud) [reply company/list-connected-companies.md]: Fixed

File: company/list-connected-companies.md [hunk 4]
```diff
@@ -3,3 +3,29 @@ description: How to list your connections with other companies
...
+{% api-method method="get" host="https://api.accp.tradecloud1.com/v2" path="/connection-search"%} 
+{% api-method-summary %} Search company connections {% endapi-method-summary %}
+{% api-method-spec %} 
+{% api-method-request %} 
+{% api-method-headers %} 
+{% api-method-parameter name="Authorization" type="string" required=true %} Authentication token {% endapi-method-parameter %} 
+{% endapi-method-headers %}
+{% api-method-query-parameters %} 
+{% api-method-parameter name="companyId" type="uuid" required=true %} Company Id {% endapi-method-parameter %}
+{% api-method-parameter name="query" type="string" required=false %} Query string {% endapi-method-parameter %}
```
[the remark this answers is not in the document]
Comment (olegtradecloud) [reply company/list-connected-companies.md]: Fixed, please verify

File: company/search-new-companies.md [hunk 5]
```diff
@@ -3,3 +3,29 @@ description: How to search for new companies to connect with
...
+{% api-method method="get" host="https://api.accp.tradecloud1.com/v2" path="/user-search/_suggest"%} 
+{% api-method-summary %} Suggest users {% endapi-method-summary %}
+{% api-method-spec %} 
+{% api-method-request %} 
+{% api-method-headers %} 
+{% api-method-parameter name="Authorization" type="string" required=true %} Authentication token {% endapi-method-parameter %} 
+{% endapi-method-headers %}
+{% api-method-query-parameters %} 
+{% api-method-parameter name="companyId" type="uuid" required=true %} Company Id {% endapi-method-parameter %}
+{% api-method-parameter name="query" type="string" required=false %} Query string {% endapi-method-parameter %}
```
[the remark this answers is not in the document]
Comment (olegtradecloud) [reply company/search-new-companies.md]: Fixed, please verify

File: company/search-new-companies.md [hunk 6]
```diff
@@ -3,3 +3,29 @@ description: How to search for new companies to connect with
 ---
 
 # Search for companies to connect
+
+You can search for new companies providing role that is opposite to your user (buyer for supplier and vice versa)
```
[the remark this answers is not in the document]
Comment (olegtradecloud) [reply company/search-new-companies.md]: Fixed

File: company/connect-to-company.md [hunk 7]
```diff
@@ -4,6 +4,34 @@ description: How to connect to another company
 
 # Connect to another company
 
-Explanation
+You can request connection to another company:
```
[the remark this answers is not in the document]
Comment (olegtradecloud) [reply company/connect-to-company.md]: Fixed

File: company/connect-to-company.md (hunk 2 above)
Comment (olegtradecloud) [reply company/connect-to-company.md]: Fixed

**Pull request merged**